### ISSUE TITLE
Routine refactor + move common into toolkit

### ISSUE THREAD

**Issue by jw-s** [vastness-io/project#4]
As more and more services are created, services drift from the desired template/framework [toolkit](https://github.com/vastness-io/toolkit).

We should make an effort to do routine maintenance on all components

**Comment by PI-Victor** [comment]
Are you referring to the `vastup` work, or in general to all components - because vastup is merely a WIP prototype at this point and can always change?

**Comment by jw-s** [comment]
@PI-Victor Sorry I wasn't clear, I meant just general to all components

**Comment by PI-Victor** [comment]
@jw-s that sounds good. will have a look through the components - do you have anything specific in mind at this point?

**Comment by jw-s** [comment]
@PI-Victor I think we should focus on exactly what options/functional comes out of the box when using our toolkit `grpc` client and so on. Distributed tracing, logging and so on.

We need to also define the process for TLS certificates etc.

**Comment by jw-s** [comment]
Do we keep the `main.go` in all services verbose like we currently have or do we provide some function/process to wrap this? If we like to keep it verbose and not hide it away then we should definitely consider a seed like project...

**Comment by PI-Victor** [comment]
yeah, `main.go` is pretty verbose, i think if we can find enough common code that is reused, then we can  move it. 

to be honest, i don't have anything against a bit of code duplication and verbosity unless there's a good enough base for a wrapper for each component.

splitting and wrapping functions can also bring some complexity that we might bring into to other components just to satisfy one of them.
